### ISSUE TITLE
"Communicable Diseases [1.2]" Massive Ticks Per Second (TPS) Slow Down (at CommunicableDiseases.HediffGiver...)

### ISSUE THREAD

**Issue by AGingersaurusRex** [cseelhoff/RimThreaded#320]
**Describe the bug**
I keep getting a "Thread # Did not finish" followed by "Exception ticking **X** (at (**x, y, z**)): System.Threading.ThreadAbortException". Where **X** is animal or pawn on the map, and **x, y, z** is the position on the map. This is causing a MASSIVE slow down in Ticks Per Second (TPS). After removing all the mods that I have added, I still have the issue.

**Error Log**
https://gist.github.com/AGingersaurusRex/6f5f41c798e9cae6799cbf68c342ef00

**Mod List**
Very Big Mod List....

Harmony | http://steamcommunity.com/sharedfiles/filedetails/?id=2009463077
BetterLoading | http://steamcommunity.com/sharedfiles/filedetails/?id=1999454301
JecsTools (Unofficial) | http://steamcommunity.com/sharedfiles/filedetails/?id=2195729899
Humanoid Alien Races 2.0 | http://steamcommunity.com/sharedfiles/filedetails/?id=839005762
RimThreaded [1.2] | http://steamcommunity.com/sharedfiles/filedetails/?id=2222907981
HugsLib | http://steamcommunity.com/sharedfiles/filedetails/?id=818773962
Vanilla Achievements Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=2288125657
Vanilla Weapons Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1814383360
Vanilla Apparel Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1814987817
Vanilla Armour Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1814988282
Vanilla Animals Expanded — Cats and Dogs | http://steamcommunity.com/sharedfiles/filedetails/?id=1823540489
Vanilla Animals Expanded — Arid Shrubland | http://steamcommunity.com/sharedfiles/filedetails/?id=1836900626
Vanilla Furniture Expanded - Security | http://steamcommunity.com/sharedfiles/filedetails/?id=1845154007
Vanilla Animals Expanded — Desert | http://steamcommunity.com/sharedfiles/filedetails/?id=1849183875
Vanilla Animals Expanded — Extreme Desert | http://steamcommunity.com/sharedfiles/filedetails/?id=1849184079
Vanilla Hair Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1888705256
Vanilla Animals Expanded — Tundra | http://steamcommunity.com/sharedfiles/filedetails/?id=1895364585
Vanilla Animals Expanded — Ice Sheet | http://steamcommunity.com/sharedfiles/filedetails/?id=1895364782
Vanilla Animals Expanded — Boreal Forest | http://steamcommunity.com/sharedfiles/filedetails/?id=1895364938
Vanilla Weapons Expanded - Quickdraw | http://steamcommunity.com/sharedfiles/filedetails/?id=1906780517
Vanilla Animals Expanded — Tropical Rainforest | http://steamcommunity.com/sharedfiles/filedetails/?id=1907283872
Vanilla Animals Expanded — Tropical Swamp | http://steamcommunity.com/sharedfiles/filedetails/?id=1907283935
Vanilla Events Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1938420742
Vanilla Furniture Expanded - Farming | http://steamcommunity.com/sharedfiles/filedetails/?id=1957158779
Vanilla Animals Expanded — Australia | http://steamcommunity.com/sharedfiles/filedetails/?id=1963681588
Vanilla Furniture Expanded - Art | http://steamcommunity.com/sharedfiles/filedetails/?id=1968134023
Vanilla Furniture Expanded - Props and Decor | http://steamcommunity.com/sharedfiles/filedetails/?id=2102143149
Vanilla Animals Expanded — Temperate Forest | http://steamcommunity.com/sharedfiles/filedetails/?id=2260502331
Vanilla Traits Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=2296404655
Vanilla Expanded Framework | http://steamcommunity.com/sharedfiles/filedetails/?id=2023507013
Better Message Placement (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=0
Better Zzzt | http://steamcommunity.com/sharedfiles/filedetails/?id=1966985422
BetterVents | http://steamcommunity.com/sharedfiles/filedetails/?id=2019372773
Better ModMismatch Window | http://steamcommunity.com/sharedfiles/filedetails/?id=1872244972
Better Vanilla Masking | http://steamcommunity.com/sharedfiles/filedetails/?id=1736114368
Better Pawn Control | http://steamcommunity.com/sharedfiles/filedetails/?id=1541460369
Better Workbench Management | http://steamcommunity.com/sharedfiles/filedetails/?id=935982361
RimUniverse - Core | http://steamcommunity.com/sharedfiles/filedetails/?id=2257043629
Vanilla Textures Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=2016436324
[HLX] ReGrowth - Core | http://steamcommunity.com/sharedfiles/filedetails/?id=2260097569
[HLX] ReGrowth - Temperate Forest Expansion | http://steamcommunity.com/sharedfiles/filedetails/?id=2261114471
[HLX] ReGrowth - Tundra Expansion | http://steamcommunity.com/sharedfiles/filedetails/?id=2107412614
[HLX] ReGrowth - Cold Bog Expansion | http://steamcommunity.com/sharedfiles/filedetails/?id=2182111148
[HLX] ReGrowth - Boreal Forest Expansion | http://steamcommunity.com/sharedfiles/filedetails/?id=2106597000
[HLX] ReGrowth - Extinct Animals Pack | http://steamcommunity.com/sharedfiles/filedetails/?id=2266685892
RimUniverse - Biomes | http://steamcommunity.com/sharedfiles/filedetails/?id=2257048351
What's That Mod | http://steamcommunity.com/sharedfiles/filedetails/?id=2258431182
Vanilla Furniture Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1718190143
[KV] RimFridge | http://steamcommunity.com/sharedfiles/filedetails/?id=1180721235
[FSF] No Default Shelf Storage | http://steamcommunity.com/sharedfiles/filedetails/?id=945085502
Ultraweave | http://steamcommunity.com/sharedfiles/filedetails/?id=916312692
Hyperweave Plantation | http://steamcommunity.com/sharedfiles/filedetails/?id=1922513504
[sd] Luciferium Production (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2019014307
Animals Logic | http://steamcommunity.com/sharedfiles/filedetails/?id=1098354593
Wild Reproduction | http://steamcommunity.com/sharedfiles/filedetails/?id=2236146674
Vanilla Plants Expanded - Succulents | http://steamcommunity.com/sharedfiles/filedetails/?id=2198652536
Avoid Friendly Fire | http://steamcommunity.com/sharedfiles/filedetails/?id=1134165362
More Vanilla Turrets 1.1 | http://steamcommunity.com/sharedfiles/filedetails/?id=2009766414
Terraform Rimworld | http://steamcommunity.com/sharedfiles/filedetails/?id=1874652867
Character Editor | http://steamcommunity.com/sharedfiles/filedetails/?id=1874644848
More Trade Ships | http://steamcommunity.com/sharedfiles/filedetails/?id=725465444
[T] MoreFloors | http://steamcommunity.com/sharedfiles/filedetails/?id=1543561093
Zoological Orbital Traders | http://steamcommunity.com/sharedfiles/filedetails/?id=1952216926
Metals Trader | http://steamcommunity.com/sharedfiles/filedetails/?id=1855069167
Automatic Night Owl | http://steamcommunity.com/sharedfiles/filedetails/?id=2056012179
Auto-Cut Blight | http://steamcommunity.com/sharedfiles/filedetails/?id=1885186534
MinifyEverything | http://steamcommunity.com/sharedfiles/filedetails/?id=872762753
Map Wide Orbital Trade Beacon | http://steamcommunity.com/sharedfiles/filedetails/?id=1839069104
Area Inclusion & Exclusion | http://steamcommunity.com/sharedfiles/filedetails/?id=2033828736
Optimization: Meats | http://steamcommunity.com/sharedfiles/filedetails/?id=1721970101
Stackable Chunks | http://steamcommunity.com/sharedfiles/filedetails/?id=1619558325
More Mechanoids | http://steamcommunity.com/sharedfiles/filedetails/?id=846222458
Biome Expansion: Redwood forest | http://steamcommunity.com/sharedfiles/filedetails/?id=1935518604
More Vanilla Biomes | http://steamcommunity.com/sharedfiles/filedetails/?id=1931453053
[CP] Rimmu-Nation - Clothing | http://steamcommunity.com/sharedfiles/filedetails/?id=1541536041
[CP] Rimmu-Nation - Weapons | http://steamcommunity.com/sharedfiles/filedetails/?id=1608498683
H.C_AnimalReskins_VarietyCoats | http://steamcommunity.com/sharedfiles/filedetails/?id=2161058753
H.C_RThrumboReskin | http://steamcommunity.com/sharedfiles/filedetails/?id=2167104490
H.C_AnimalReskins_Complete | http://steamcommunity.com/sharedfiles/filedetails/?id=2146766912
H.C_AnimalReskins_4.1 | http://steamcommunity.com/sharedfiles/filedetails/?id=2114403620
H.C_AnimalReskins_4 | http://steamcommunity.com/sharedfiles/filedetails/?id=2112514903
[FSF] Improved Vanometric Power Cells | http://steamcommunity.com/sharedfiles/filedetails/?id=1212274603
[FSF] Growable Mushrooms | http://steamcommunity.com/sharedfiles/filedetails/?id=1180192852
[FSF] Growable Ambrosia | http://steamcommunity.com/sharedfiles/filedetails/?id=1349805309
Biomes! Core | http://steamcommunity.com/sharedfiles/filedetails/?id=2038000893
GloomyFurniture | http://steamcommunity.com/sharedfiles/filedetails/?id=1558635181
[FSF] Indoor Tree Farms | http://steamcommunity.com/sharedfiles/filedetails/?id=1515299608
Dubs Bad Hygiene | http://steamcommunity.com/sharedfiles/filedetails/?id=836308268
Colony Manager | http://steamcommunity.com/sharedfiles/filedetails/?id=715565262
Vanilla Books Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=2193152410
Vanilla Brewing Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=2186560858
Vanilla Brewing Expanded - Coffees and Teas | http://steamcommunity.com/sharedfiles/filedetails/?id=2275449762
[FSF] Complex Jobs | http://steamcommunity.com/sharedfiles/filedetails/?id=2069684319
[FSF] Better Spike Traps | http://steamcommunity.com/sharedfiles/filedetails/?id=1459119465
Solar Flares Affect Mechanoids | http://steamcommunity.com/sharedfiles/filedetails/?id=1906174567
Rimworld: Spartan Foundry | http://steamcommunity.com/sharedfiles/filedetails/?id=1695609511
More Filters | http://steamcommunity.com/sharedfiles/filedetails/?id=1911734422
More Shelves | http://steamcommunity.com/sharedfiles/filedetails/?id=1659929907
More Batteries | http://steamcommunity.com/sharedfiles/filedetails/?id=747397788
More Planning | http://steamcommunity.com/sharedfiles/filedetails/?id=881100382
Super Soil | http://steamcommunity.com/sharedfiles/filedetails/?id=823626435
Growable Neutroamine | http://steamcommunity.com/sharedfiles/filedetails/?id=1956517495
Chatting on Comms | http://steamcommunity.com/sharedfiles/filedetails/?id=2058332610
Stabilize | http://steamcommunity.com/sharedfiles/filedetails/?id=2023407836
Traders Have Money | http://steamcommunity.com/sharedfiles/filedetails/?id=2022959333
Run and Hide (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2022480344
PawnTargetFix | http://steamcommunity.com/sharedfiles/filedetails/?id=2014789938
Quarry 1.1 | http://steamcommunity.com/sharedfiles/filedetails/?id=2007576583
Simple Search Bar | http://steamcommunity.com/sharedfiles/filedetails/?id=1827546987
Mortar Accuracy | http://steamcommunity.com/sharedfiles/filedetails/?id=1729446857
Everybody Gets One | http://steamcommunity.com/sharedfiles/filedetails/?id=1687566130
Graphics Settings+ | http://steamcommunity.com/sharedfiles/filedetails/?id=1678847247
FrameRateControl | http://steamcommunity.com/sharedfiles/filedetails/?id=1591142767
[KV] Consolidated Traits | http://steamcommunity.com/sharedfiles/filedetails/?id=1507384362
Cut plants before building | http://steamcommunity.com/sharedfiles/filedetails/?id=1539025677
Tables+ | http://steamcommunity.com/sharedfiles/filedetails/?id=963265186
Real Silver Weight | http://steamcommunity.com/sharedfiles/filedetails/?id=2145184299
Destroy Anything | http://steamcommunity.com/sharedfiles/filedetails/?id=2132868354
Lag Free Lamps | http://steamcommunity.com/sharedfiles/filedetails/?id=2110261745
Deep Scanner Settings | http://steamcommunity.com/sharedfiles/filedetails/?id=2088979675
Bulk Smelting | http://steamcommunity.com/sharedfiles/filedetails/?id=2060240608
Metal Don't Burn | http://steamcommunity.com/sharedfiles/filedetails/?id=2022726345
Room Size Tolerance | http://steamcommunity.com/sharedfiles/filedetails/?id=2018432252
Animal Medical Bed | http://steamcommunity.com/sharedfiles/filedetails/?id=2016958187
Damage Indicators [1.2] | http://steamcommunity.com/sharedfiles/filedetails/?id=2016331497
Invisible Conduit | http://steamcommunity.com/sharedfiles/filedetails/?id=2013416365
Craftsmanship | http://steamcommunity.com/sharedfiles/filedetails/?id=2010945849
No Job Authors | http://steamcommunity.com/sharedfiles/filedetails/?id=2009825774
CM Color Coded Mood Bar [1.1+] | http://steamcommunity.com/sharedfiles/filedetails/?id=2006605356
Trade Ships Drop Spot | http://steamcommunity.com/sharedfiles/filedetails/?id=1969732297
Carry Capacity Fixed | http://steamcommunity.com/sharedfiles/filedetails/?id=1906760965
Vanilla Furniture Expanded - Production | http://steamcommunity.com/sharedfiles/filedetails/?id=1880253632
[WD] Reinforced Doors | http://steamcommunity.com/sharedfiles/filedetails/?id=1857597307
RimHUD | http://steamcommunity.com/sharedfiles/filedetails/?id=1508850027
Blighted Alert | http://steamcommunity.com/sharedfiles/filedetails/?id=1494937292
Wall Vitals Monitor | http://steamcommunity.com/sharedfiles/filedetails/?id=1334289344
Architect Icons | http://steamcommunity.com/sharedfiles/filedetails/?id=1195427067
ResearchPal | http://steamcommunity.com/sharedfiles/filedetails/?id=946390822
Fahrenheit and Celsius | http://steamcommunity.com/sharedfiles/filedetails/?id=937759575
Animal Tab | http://steamcommunity.com/sharedfiles/filedetails/?id=712141500
Medical Tab | http://steamcommunity.com/sharedfiles/filedetails/?id=715565817
Misc. Training | http://steamcommunity.com/sharedfiles/filedetails/?id=717575199
Wall Light | http://steamcommunity.com/sharedfiles/filedetails/?id=1423699208
RimCore | http://steamcommunity.com/sharedfiles/filedetails/?id=2268869457
Wall Sun Lamp | http://steamcommunity.com/sharedfiles/filedetails/?id=1825653111
WallStuff | http://steamcommunity.com/sharedfiles/filedetails/?id=1994340640
Vanilla Hair Expanded - Beards | http://steamcommunity.com/sharedfiles/filedetails/?id=2187520791
Silent Doors (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2012447929
Reinforced Walls | http://steamcommunity.com/sharedfiles/filedetails/?id=1929389479
Vanilla Fishing Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1914064942
Vanilla Factions Expanded - Insectoids | http://steamcommunity.com/sharedfiles/filedetails/?id=2149755445
Beautiful Outdoors | http://steamcommunity.com/sharedfiles/filedetails/?id=2011794898
Veinminer R1.1 | http://steamcommunity.com/sharedfiles/filedetails/?id=1541978411
Selectable Sculpture Graphic | http://steamcommunity.com/sharedfiles/filedetails/?id=1537873717
[SYR] Harvest Yield | http://steamcommunity.com/sharedfiles/filedetails/?id=1461790308
Vanilla Furniture Expanded - Medical Module | http://steamcommunity.com/sharedfiles/filedetails/?id=1718191613
More Sculpture | http://steamcommunity.com/sharedfiles/filedetails/?id=1612316880
Un-Limited | http://steamcommunity.com/sharedfiles/filedetails/?id=1668141382
Search and Destroy | http://steamcommunity.com/sharedfiles/filedetails/?id=1467764609
Give Me Meat | http://steamcommunity.com/sharedfiles/filedetails/?id=2020705229
OgreStack | http://steamcommunity.com/sharedfiles/filedetails/?id=1447140290
Vanilla Furniture Expanded - Spacer Module | http://steamcommunity.com/sharedfiles/filedetails/?id=2028381079
Diseases+ | http://steamcommunity.com/sharedfiles/filedetails/?id=2256624398
SRTS Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1845423808
Craftable Synthetics Continued [1.0][1.1][1.2] | http://steamcommunity.com/sharedfiles/filedetails/?id=2016498352
Glass+Lights | http://steamcommunity.com/sharedfiles/filedetails/?id=826153738
Dubs Mint Menus | http://steamcommunity.com/sharedfiles/filedetails/?id=1446523594
Snap Out! | http://steamcommunity.com/sharedfiles/filedetails/?id=1319782555
Enhanced Combat Sounds | http://steamcommunity.com/sharedfiles/filedetails/?id=2095586726
More Linkables | http://steamcommunity.com/sharedfiles/filedetails/?id=1103809207
Dub's Paint Shop | http://steamcommunity.com/sharedfiles/filedetails/?id=1579516669
The Birds and the Bees | http://steamcommunity.com/sharedfiles/filedetails/?id=751049133
Area Unlocker | http://steamcommunity.com/sharedfiles/filedetails/?id=725153576
Mad Skills | http://steamcommunity.com/sharedfiles/filedetails/?id=731111514
P-Music | http://steamcommunity.com/sharedfiles/filedetails/?id=725130005
Level Up! | http://steamcommunity.com/sharedfiles/filedetails/?id=1701592470
Tilled Soil | http://steamcommunity.com/sharedfiles/filedetails/?id=725747149
[FSF] Growable Grass | http://steamcommunity.com/sharedfiles/filedetails/?id=817350064
QualitySurgeon | http://steamcommunity.com/sharedfiles/filedetails/?id=857350593
Camera+ | http://steamcommunity.com/sharedfiles/filedetails/?id=867467808
[KV] Don't stop the music | http://steamcommunity.com/sharedfiles/filedetails/?id=1210224966
Fixable Mood Debuffs Alert | http://steamcommunity.com/sharedfiles/filedetails/?id=1307316672
Safely Hidden Away | http://steamcommunity.com/sharedfiles/filedetails/?id=1322196379
Share The Load | http://steamcommunity.com/sharedfiles/filedetails/?id=1356838246
Pharmacist | http://steamcommunity.com/sharedfiles/filedetails/?id=1365242717
More Harvest Designators! | http://steamcommunity.com/sharedfiles/filedetails/?id=1541250497
Show Draftees Weapon | http://steamcommunity.com/sharedfiles/filedetails/?id=1690978457
Vanilla Weapons Expanded - Laser | http://steamcommunity.com/sharedfiles/filedetails/?id=1989352844
Bulk Stonecutting | http://steamcommunity.com/sharedfiles/filedetails/?id=2021425587
Leaf Like Life | http://steamcommunity.com/sharedfiles/filedetails/?id=2024017049
Power Indicators | http://steamcommunity.com/sharedfiles/filedetails/?id=899714534
[KV] Trading Spot | http://steamcommunity.com/sharedfiles/filedetails/?id=1180719658
[KV] Save Storage, Outfit, Crafting, Drug, & Operation Settings | http://steamcommunity.com/sharedfiles/filedetails/?id=1180718516
Reasonable Components | http://steamcommunity.com/sharedfiles/filedetails/?id=1542915888
Recipe icons | http://steamcommunity.com/sharedfiles/filedetails/?id=1616643195
Antigrain | http://steamcommunity.com/sharedfiles/filedetails/?id=2016911592
Archotech Expanded | http://steamcommunity.com/sharedfiles/filedetails/?id=1442734859
Micro Designations | http://steamcommunity.com/sharedfiles/filedetails/?id=1877972690
[WD] Soil Contrast | http://steamcommunity.com/sharedfiles/filedetails/?id=1561838700
[KV] Faction Control | http://steamcommunity.com/sharedfiles/filedetails/?id=1509102551
Pawn Badge Fan Fork | http://steamcommunity.com/sharedfiles/filedetails/?id=2112663412
Pawn Badge - US Army Officer Ranks | http://steamcommunity.com/sharedfiles/filedetails/?id=2213603313
Pawn Badge - (MISC) Job Icons | http://steamcommunity.com/sharedfiles/filedetails/?id=1734233154
Pawn Badge - Assorted Job Icons | http://steamcommunity.com/sharedfiles/filedetails/?id=2184581646
Forest Critters | http://steamcommunity.com/sharedfiles/filedetails/?id=1970508422
LWM's Deep Storage | http://steamcommunity.com/sharedfiles/filedetails/?id=1617282896
[WD] Realistic Darkness | http://steamcommunity.com/sharedfiles/filedetails/?id=1531882210
[JPT] Soft Warm Beds | http://steamcommunity.com/sharedfiles/filedetails/?id=1713295858
Caravan Lag Eliminator | http://steamcommunity.com/sharedfiles/filedetails/?id=2248500261
Interaction Bubbles | http://steamcommunity.com/sharedfiles/filedetails/?id=1516158345
Numbers (patched) 1.2 | http://steamcommunity.com/sharedfiles/filedetails/?id=2255104042
A Sloth Mod | http://steamcommunity.com/sharedfiles/filedetails/?id=2253087891
Jernalk's Skeletal Legion (continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2255092108
More Thrumbos | http://steamcommunity.com/sharedfiles/filedetails/?id=1789924054
[WD] Antibiotics And Painkillers | http://steamcommunity.com/sharedfiles/filedetails/?id=1692811928
Blueprints | http://steamcommunity.com/sharedfiles/filedetails/?id=708455313
Work Tab | http://steamcommunity.com/sharedfiles/filedetails/?id=725219116
MendAndRecycle | http://steamcommunity.com/sharedfiles/filedetails/?id=735241897
Advanced Power Plus | http://steamcommunity.com/sharedfiles/filedetails/?id=2032409628
Archotech Expanded + Vanilla Art Expanded [Craftable Archotech Sculpures] Patch | http://steamcommunity.com/sharedfiles/filedetails/?id=2139688225
Archotech Expanded + Cyber Fauna Patch | http://steamcommunity.com/sharedfiles/filedetails/?id=2139689653
Archotech+ (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2270493883
Archotech Expanded + Archotech+ Patch | http://steamcommunity.com/sharedfiles/filedetails/?id=2139690498
Stuffed Security | http://steamcommunity.com/sharedfiles/filedetails/?id=2280581106
Craftable Cores | http://steamcommunity.com/sharedfiles/filedetails/?id=1985991383
Glitterworld medicine craftable | http://steamcommunity.com/sharedfiles/filedetails/?id=1541794953
Advanced Parka | http://steamcommunity.com/sharedfiles/filedetails/?id=1677752791
Accurate Mortars | http://steamcommunity.com/sharedfiles/filedetails/?id=2034585850
Advanced Mortar (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2023771298
No Carried Food | http://steamcommunity.com/sharedfiles/filedetails/?id=1723472015
Drug Policy Fix (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2195390245
Stuffed Workbenches | http://steamcommunity.com/sharedfiles/filedetails/?id=940971981
Marine Armor Materials | http://steamcommunity.com/sharedfiles/filedetails/?id=1638630888
Spartan Foundry Warcraft (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2024346916
Use Minified Buildings | http://steamcommunity.com/sharedfiles/filedetails/?id=2185809028
Doors Expanded (Dev) | http://steamcommunity.com/sharedfiles/filedetails/?id=2195733555
Craft AI Core | http://steamcommunity.com/sharedfiles/filedetails/?id=2189950967
[WD] Expanded Floors | http://steamcommunity.com/sharedfiles/filedetails/?id=1817862464
[WD] Expanded Floors (Dropdown Menu) | http://steamcommunity.com/sharedfiles/filedetails/?id=1817883612
[LTO] Legacy Terrain Overhaul | http://steamcommunity.com/sharedfiles/filedetails/?id=2284769535
Bionic icons | http://steamcommunity.com/sharedfiles/filedetails/?id=1677616980
Cyber Fauna 1.2 | http://steamcommunity.com/sharedfiles/filedetails/?id=1548649032
Life Support Continued [1.1][1.2] | http://steamcommunity.com/sharedfiles/filedetails/?id=2213696082
Designator Shapes | http://steamcommunity.com/sharedfiles/filedetails/?id=1235181370
[SYR] Neuter | http://steamcommunity.com/sharedfiles/filedetails/?id=1501449584
[SYR] More Slaves | http://steamcommunity.com/sharedfiles/filedetails/?id=2021510577
Extra Walls and Floors | http://steamcommunity.com/sharedfiles/filedetails/?id=1882432197
Export Agency | http://steamcommunity.com/sharedfiles/filedetails/?id=1467209473
Map Reroll | http://steamcommunity.com/sharedfiles/filedetails/?id=761315214
Custom Year and Speed | http://steamcommunity.com/sharedfiles/filedetails/?id=2093902544
Climate Cycle++ (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2024296023
Custom Climate Cycle | http://steamcommunity.com/sharedfiles/filedetails/?id=1684902496
Dismiss Trader (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2078518511
Skilled Stonecutting | http://steamcommunity.com/sharedfiles/filedetails/?id=1207390617
Change Caravan Loadout | http://steamcommunity.com/sharedfiles/filedetails/?id=2139318762
Balanced Eclipse and Solar Flare | http://steamcommunity.com/sharedfiles/filedetails/?id=1254969827
[RH] Faction: Task Force 141 | http://steamcommunity.com/sharedfiles/filedetails/?id=1720751798
[RH] Faction: UAC | http://steamcommunity.com/sharedfiles/filedetails/?id=2130954176
[RH] Faction: War Mongrels | http://steamcommunity.com/sharedfiles/filedetails/?id=2130880638
[RH] Faction: Coalition | http://steamcommunity.com/sharedfiles/filedetails/?id=2130877773
[RH] Faction: Last Man Contingent | http://steamcommunity.com/sharedfiles/filedetails/?id=1864031417
[RH] Faction: Elite Crew | http://steamcommunity.com/sharedfiles/filedetails/?id=1864031916
[RH] Faction: VOID | http://steamcommunity.com/sharedfiles/filedetails/?id=2130955778
[RH] Faction: Allegiance | http://steamcommunity.com/sharedfiles/filedetails/?id=2130878466
RenameColony | http://steamcommunity.com/sharedfiles/filedetails/?id=1359645236
Rim War | http://steamcommunity.com/sharedfiles/filedetails/?id=2222935097
[NL] Dynamic Portraits | http://steamcommunity.com/sharedfiles/filedetails/?id=2253730555
[NL] Custom Portraits | http://steamcommunity.com/sharedfiles/filedetails/?id=1569605867
Tate's Tiny Furniture Mod | http://steamcommunity.com/sharedfiles/filedetails/?id=2247819099
BeautyFloors (Continued) | http://steamcommunity.com/sharedfiles/filedetails/?id=2054580861
Statue of Colonist | http://steamcommunity.com/sharedfiles/filedetails/?id=1539031321
SF More Steel from Slag | http://steamcommunity.com/sharedfiles/filedetails/?id=1558034969
Realistic Rooms | http://steamcommunity.com/sharedfiles/filedetails/?id=848972794
[FSF] More Quests | http://steamcommunity.com/sharedfiles/filedetails/?id=1975779192
SimplyMoreFloors | http://steamcommunity.com/sharedfiles/filedetails/?id=1639883955
[KV] More Trait Slots | http://steamcommunity.com/sharedfiles/filedetails/?id=1508292711
Pawn Rules | http://steamcommunity.com/sharedfiles/filedetails/?id=1499843448
Communicable Diseases [1.2] | http://steamcommunity.com/sharedfiles/filedetails/?id=2016577520
Better Infestations 1.2 | http://steamcommunity.com/sharedfiles/filedetails/?id=1319614331
Bandage Wraps | http://steamcommunity.com/sharedfiles/filedetails/?id=2202226653
Dress Patients | http://steamcommunity.com/sharedfiles/filedetails/?id=1667901146
Questionable Ethics Enhanced | http://steamcommunity.com/sharedfiles/filedetails/?id=1787359789
RimTraits - Vanilla Trait Colours | http://steamcommunity.com/sharedfiles/filedetails/?id=2244594116
RimTraits - General Traits | http://steamcommunity.com/sharedfiles/filedetails/?id=2206957172
Sparkling Worlds Addon - Mech Serum Crafting - Standalone Addon | http://steamcommunity.com/sharedfiles/filedetails/?id=1511462447
YIMBY Arts - Ambient Jams | http://steamcommunity.com/sharedfiles/filedetails/?id=731638290
Dire Raids | http://steamcommunity.com/sharedfiles/filedetails/?id=1542379675
Carcinoma spreads | http://steamcommunity.com/sharedfiles/filedetails/?id=1206669918
Thrumbo Extension | http://steamcommunity.com/sharedfiles/filedetails/?id=859466666
Supply and Demand | http://steamcommunity.com/sharedfiles/filedetails/?id=1511375007
Animal Food Restrictions | http://steamcommunity.com/sharedfiles/filedetails/?id=1547017052
No Tremendous saddness with death of prisoner | http://steamcommunity.com/sharedfiles/filedetails/?id=1831092867
More Descriptive Words and Names | http://steamcommunity.com/sharedfiles/filedetails/?id=2180384246
KanbanStockpile | http://steamcommunity.com/sharedfiles/filedetails/?id=2287142613
Compact Hediffs | http://steamcommunity.com/sharedfiles/filedetails/?id=2031734067

**Comment by Niilo007** [comment]
The error at least was caused by "Communicable Diseases [1.2]"
I am adding this mod to the incompatible list

**Comment by Eldulac** [comment]
**[Tested on RT 1.3.2]**

I tested this issue on a couple of fresh games, with only RT and "Communicable Diseases [1.2]" loaded. The tests were done on temperate forest and rainforest biomes. I spawned dozens of pawns and animals, and then spread various diseases among them in series. After more than an in-game week of speed 4 testing, there were no issues with tracking filth or communicating diseases.

This issue has likely been caused by load order problems and incompatibilities, of which I spotted several in the mod list. The biggest of which was RT not being at the bottom of the list.
I have added a couple of test save games here, in case further testing is required.

[Communicable Diseases Issue 2.zip](https://github.com/cseelhoff/RimThreaded/files/6156888/Communicable.Diseases.Issue.2.zip)
[Communicable diseases Issue.zip](https://github.com/cseelhoff/RimThreaded/files/6156891/Communicable.diseases.Issue.zip)

**Conclusion**
There is no indication of problems between "Communicable Diseases [1.2]" and RT. The problem described in the report is likely caused by load order issues and incompatibilities, or has already been fixed.
